<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CampaignSelectionClass</class>
 <widget class="QWidget" name="CampaignSelectionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CampaignSelection</string>
  </property>
  <widget class="QListView" name="list_campaign">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>341</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_startGame">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_back">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>230</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
